src/main/java/org/phoebus/channelfinder/ElasticConfig.java

```diff
@@ -91,8 +95,10 @@ public class ElasticConfig implements ServletContextListener {
     private String ES_PROPERTY_INDEX;
     @Value("${elasticsearch.channel.index:channelfinder}")
     private String ES_CHANNEL_INDEX;
-    @Value("${elasticsearch.query.size}")
+    @Value("${elasticsearch.query.size:10000}")
     private int ES_QUERY_SIZE;
+    @Value("${elasticsearch.maxResultWindow.size:10000}")
```

Comment:
hmm... 
I guess we could, since this setting updates the actual index settings I thought it would be safer to have it explicitly defined... I am happy to be convinced the other way if you think I am being over cautious.